fastembed/rerank/cross_encoder/onnx_text_model.py

```diff
@@ -45,8 +45,10 @@ def _load_onnx_model(
         self.tokenizer, _ = load_tokenizer(model_dir=model_dir)
         assert self.tokenizer is not None
 
-    def tokenize(self, pairs: list[tuple[str, str]], **_: Any) -> list[Encoding]:
-        return self.tokenizer.encode_batch(pairs)  # type: ignore[union-attr]
+    def tokenize(self, pairs: list[tuple[str, str]], **kwargs: Any) -> list[Encoding]:
+        if self.tokenizer is None:
+            raise RuntimeError("Tokenizer not initialized")
```

Comment:
we can probably load it here if it hasn't been initialized yet, would be perfect toload only tokenizer, need to check whether we can load it separately or how difficult it to do so